src/DataShare/DataShare/Models/ModelExtensions.cs

```diff
@@ -0,0 +1,545 @@
+﻿// ----------------------------------------------------------------------------------
```

Comment:
Remove the file above: src/DataShare/DataShare/Models/ModelExtensions.cs

Reply:
where should we move the ToPsObject() functions ?